saleor/graphql/order/types.py

```diff
@@ -1029,6 +1038,23 @@ def wrap_shipping_method_with_channel_context(data):
             wrap_shipping_method_with_channel_context
         )
 
+    @staticmethod
+    @traced_resolver
+    def resolve_delivery_method(root: models.Order, info):
+        if root.shipping_method_id:
+            return Order.resolve_shipping_method(root, info)
+        if root.collection_point_id:
+            collection_point = WarehouseByIdLoader(info.context).load(
+                root.collection_point_id
+            )
+            return collection_point
+        return None
+
+    @staticmethod
+    @traced_resolver
```

Comment:
In PR #7661 we drop tracing for simple resolvers. 
```suggestion
```

Reply:
Done.

Reply:
I hope it's done now, my apologise ;( 🤦